aiida/restapi/common/utils.py

```diff
@@ -8,13 +8,15 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """ Util methods """
-import urllib.parse
 from datetime import datetime, timedelta
+import urllib.parse
+from wrapt import decorator
```

Comment:
minor nitpick: this is third-party library and so should go after `from flask.json` (surprised this is not picked up by `pylint`)

Reply:
It wasn't picked up by either `pylint` nor the pre-commit `prospector` (which usually _always_ catches these things).

Reply:
And `flask` is a third-party library as well ... right?

Reply:
yes, but the `w` comes after the `f` in the alphabet :) As I said, nitpick ...

Reply:
🙄 ...